apps/cowswap-frontend/src/modules/balancesAndAllowances/updaters/CommonPriorityBalancesAndAllowancesUpdater.tsx

```diff
@@ -48,20 +64,29 @@ export function CommonPriorityBalancesAndAllowancesUpdater(): ReactNode {
     }
   }, [account, priorityTokenCount])
 
+  const { enableBffBalancesFeePercentage } = useFeatureFlags()
+  const isBffFailed = useIsBffFailed()
+  const isBffEnabled = shouldApplyBffBalances(account, enableBffBalancesFeePercentage) && !isBffFailed
+  const pendingOrdersCount = usePendingOrdersCount(sourceChainId, balancesAccount)
+
   return (
     <>
-      <PriorityTokensUpdater
-        // We can and should save one RPC call at the very beginning
-        // Since regular BalancesAndAllowancesUpdater will update all tokens (including priority tokens)
-        // We can skip first update for PriorityTokensUpdater
-        account={skipFirstPriorityUpdate ? undefined : balancesAccount}
-        chainId={sourceChainId}
-        tokenAddresses={priorityTokenAddressesAsArray}
-      />
+      {!isBffEnabled ? (
```

Comment:
Maybe we should keep `PriorityTokensUpdater` always enabled?
It doesn't make heavy multicalls, so we can afford that.